code/modules/clothing/shoes/miscellaneous.dm

```diff
@@ -21,15 +21,15 @@
 	can_cut_open = 1
 	icon_state = "jackboots"
 	item_state = "jackboots"
-	armor = list(melee = 50, bullet = 50, laser = 50, energy = 25, bomb = 50, bio = 10, rad = 0)
+	armor = list(melee = 25, bullet = 25, laser = 25, energy = 25, bomb = 50, bio = 10, rad = 0)

```

Comment:
I'd guess if you hit their foot (maybe leg, since they're boots?) it'd be a factor